examples/crm/src/App.tsx

```diff
@@ -45,6 +46,7 @@ const App = () => (
         <Resource name="deals" {...deals} />
         <Resource name="contacts" {...contacts} />
         <Resource name="companies" {...companies} />
+        <Resource name="sales" {...sales} />
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** Shouldn't we prevent access to the resource directly? rather than displaying an empty page?

In any case, if we don't block, we'd need a message rather than an empty page.‏